scipy/signal/filter_design.py

```diff
@@ -161,6 +173,94 @@ def freqs(b, a, worN=None, plot=None):
     return w, h
 
 
+def freqs_zpk(z, p, k, worN=None, plot=None):
```

Comment:
I'm fine with leaving it in for consistency or removing it (as I wish it weren't in `freqs` in the first place)